config/config.go

```diff
@@ -164,6 +164,7 @@ func OpenAIConfigFromServiceConfig(serviceConfig llm.ServiceConfig, enabledNativ
 		StreamingTimeout:   streamingTimeout,
 		SendUserID:         serviceConfig.SendUserID,
 		UseResponsesAPI:    serviceConfig.UseResponsesAPI,
-		EnabledNativeTools: enabledNativeTools,
+		EnabledNativeTools: botConfig.EnabledNativeTools,
+		BotConfig:          botConfig,
```

Comment:
Same here probably want extract the individual parameters.

Reply:
UPdated